packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/Result.tsx

```diff
@@ -17,6 +18,9 @@ function ResultStep({ form }: WizardStepProps): JSX.Element {
   }, [form]);
 
   const notifications = publicodesContext.getNotifications();
+  const references = publicodesContext.getReferences();
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Not needed anymore, should be removed. It was here for the `A11yLink` component which has been move to typescript.